### ISSUE TITLE
RGB Broken Rog Ally

### ISSUE THREAD

**Issue by Bladen123** [ublue-os/bazzite#1343]
### Describe the bug

So this makes me upset now , i installed latest bazziteOS and after i Boot Windows my Rainbow RGB is Gone Again after all this Updates u Guys did arent u able to fix this ? Now there was a fix to downgrade to a earlyer Version but even that doesnt Work anymore ON new BazziteOS . Please give me the fix for this . And i would suggest u mention this Problem in your Download since this is litteraly F***ed UP.  I want my Rainbow RGB Back please .its funny because cachyOS Handheld and any Other SteamOS fork doesnt ruin our Asus RGB only this bazziteOS and this is a very old good know Problem and still is Here . 

### What did you expect to happen?

I expected 2 + 2 to equal 4, but instead 2 + 2 equaled 6!

**Comment by KyleGospo** [comment]
We do this for free, do not open another issue until you have read our code of conduct.

**Comment by Bladen123** [comment]
What are U meaning ? I Just want to get my RGB ON Windows Back !!!!! How can i rebase rollback to old Version from 20240427 thats what fixed the RGB Back to Work ON Windows

**Comment by Harithz001** [comment]
Has anyone found a fix to this issue? This is such a bummer as I'm receiving my Ally X in a few days time yet I am stuck, unable to re-sell my original Ally as the rainbow led is broken thanks to bazzite. I'm sure there are many others in similar predicament. Dear devs, please assist. Thank you.

**Comment by KyleGospo** [comment]
> Has anyone found a fix to this issue? This is such a bummer as I'm receiving my Ally X in a few days time yet I am stuck, unable to re-sell my original Ally as the rainbow led is broken thanks to bazzite. I'm sure there are many others in similar predicament. Dear devs, please assist. Thank you.

Please see https://github.com/ublue-os/bazzite/issues/1153 for solutions
